instrumentation/opentelemetry-instrumentation-mysqlclient/tests/test_mysqlclient_integration.py

```diff
@@ -96,6 +96,42 @@ def test_instrument_connection(self, mock_connect):
         spans_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans_list), 1)
 
+    @mock.patch("opentelemetry.instrumentation.dbapi.instrument_connection")
```

Comment:
Do any of these tests check that the SQL string is actually sent with the proper comment?

Reply:
Nope, as I assumed [test_dbapi_integration](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-dbapi/tests/test_dbapi_integration.py) would cover it as long as mysqlclient instrumentor passes the kwargs. But more tests especially for integration would be good, so I'll work on that!

Reply:
Test writing helped me find a bug so thanks 😄 I've put it in a separate PR here: https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3027

Reply:
I've added more tests, but they'll fail until https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3027 is approved.

Reply:
Merged